src/domain/hub/utils/HubUtils.ts

```diff
@@ -0,0 +1,13 @@
+const CONCATENATED_UUIDS_REGEX = /^([a-fA-F\-0-9]+)\/([a-fA-F\-0-9]+)$/;
+
+/**
+ * Handles concatenated {OrgId}/{HubId} identifiers coming from the Role Service
+ * @param hubId HubId or OrganizationId/HubId
+ * @returns HubId
+ */
+export const handleHubId = (hubId: string) => {
```

Comment:
I like to start function names with handle if they handle actions (handleSubmit, handleSave, etc.) otherwise it is nicer to have a more clearer name that explains what the function does. In this case, extractHubId or something similar.